Arrowgene.Ddon.GameServer/Handler/QuestGetRewardBoxListHandler.cs

```diff
@@ -17,13 +17,14 @@
 
 namespace Arrowgene.Ddon.GameServer.Handler
 {
-    public class QuestGetRewardBoxHandler : PacketHandler<GameClient>
+    public class QuestGetRewardBoxListHandler : PacketHandler<GameClient>
     {
-        private static readonly ServerLogger Logger = LogProvider.Logger<ServerLogger>(typeof(QuestGetRewardBoxHandler));
+        private static readonly ServerLogger Logger = LogProvider.Logger<ServerLogger>(typeof(QuestGetRewardBoxListHandler));
 
-
-        public QuestGetRewardBoxHandler(DdonGameServer server) : base(server)
+        private DdonGameServer _Server;
+        public QuestGetRewardBoxListHandler(DdonGameServer server) : base(server)
         {
+            _Server = server;
```

Comment:
If you extend GameStructurePacketHandler (or preferrably GameRequestPacketHandler) you don't need to store the DdonGameServer instance.

Reply:
This one has an empty request packet so I didn't implement it. You want me to make a request packet for this still?

Reply:
A packet with no concents you mean? It wouldnt be the first time we have those. It'd be better to have it implemented regardless for consistency

Reply:
Actually, a lot of requests don't have it implemented which is where I copied this pattern from :)

Reply:
Done.